app/code/Magento/Customer/view/adminhtml/ui_component/customer_group_form.xml

```diff
@@ -15,11 +15,27 @@
     </argument>
     <settings>
         <buttons>
-            <button class="Magento\Customer\Block\Adminhtml\Group\Edit\BackButton" name="back"/>
+            <button name="back">
+                <url path="*/*/"/>
+                <class>back</class>
+                <label translate="true">Back</label>
+            </button>
             <button class="Magento\Customer\Block\Adminhtml\Group\Edit\DeleteButton" name="delete"/>
-            <button class="Magento\Customer\Block\Adminhtml\Group\Edit\ResetButton" name="reset"/>
-            <button class="Magento\Customer\Block\Adminhtml\Group\Edit\SaveButton" name="save"/>
-            <button class="Magento\Customer\Block\Adminhtml\Group\Edit\SaveAndContinueButton" name="save_and_continue"/>
+            <button name="reset">
+                <url path="*/*/*"/>
+                <class>reset</class>
+                <label translate="true">Reset</label>
+            </button>
+            <button name="save">
+                <url path="*/*/save"/>
+                <class>save primary</class>
+                <label translate="true">Save Customer Group</label>
+            </button>
+            <button name="save_and_continue">
```

Comment:
I don't think that configuration for save_and continue is correct.

The reflection of the array that was returned by getButtonData() method:

        return [	
            'label' => __('Save and Continue Edit'),	
            'class' => 'save',	
            'data_attribute' => [	
                'mage-init' => [	
                    'button' => ['event' => 'saveAndContinueEdit'],	
                ],	
            ],	
            'sort_order' => 80,	
        ];

Is the following XML:

    <item name="save_and_continue" xsi:type="array">
        <item name="label" xsi:type="string" translate="true">Save and Continue Edit</item>
        <item name="class" xsi:type="string">save</item>
        <item name="data_attribute" xsi:type="array">
            <item name="mage-init" xsi:type="array">
                <item name="button" xsi:type="array">
                    <item name="event" xsi:type="string">saveAndContinueEdit</item>
                </item>
            </item>
        </item>
        <item name="sort_order" xsi:type="number">80</item>
    </item>

Reply:
@sivaschenko for the save_and_continue there are two option.

1. the one I have implemented
2. Second one is through a class

The solution you gave can't be used for Magento 2.2 or 2.3

Reply:
@lewisvoncken the snippet I provided is simply a translation of the array you used in Button class to the XML. I don't think the URL "*/*/save/back/1" will properly work in this case.